deploy/compose/.env.example

```
@@ -0,0 +1,31 @@
+POSTGRES_PASSWORD=replace-with-a-random-database-password
+OPENTAG_BOOTSTRAP_ORGANIZATION_ID=org_local
+OPENTAG_BOOTSTRAP_ORGANIZATION_NAME=Local OpenTag
+OPENTAG_BOOTSTRAP_PAIRING_TOKEN=replace-with-at-least-16-random-characters
+OPENTAG_FENCING_TOKEN_SECRET=replace-with-at-least-32-independent-random-characters
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPENTAG_FENCING_TOKEN_SECRET key should go before the POSTGRES_PASSWORD key

(UnorderedKey)